examples/lighting-app/nrfconnect/configuration/nrf52840dk_nrf52840/pm_static.yml

```diff
@@ -7,32 +7,32 @@ mcuboot_pad:
     size: 0x200
 app:
     address: 0xc200
-    size: 0xf1e00
+    size: 0xebe00
 mcuboot_primary:
     orig_span: &id001
         - mcuboot_pad
         - app
     span: *id001
     address: 0xc000
-    size: 0xf2000
+    size: 0xec000
```

Comment:
Isn't `span` attribute supposed to determine the address and size autoamtically?

Reply:
I guess not, at least if you will not assign address and size values for each partition it's going to crash at:
```  File "/var/ncs/nrf/scripts/partition_manager.py", line 682, in calculate_end_address
    pm_config[part]['end_address'] = pm_config[part]['address'] + pm_config[part]['size']```

Reply:
Ok, so I don't understand what's the point of the span :)

Reply:
Neither do I, but it also crashes when removing span field. Partition manager IMHO is ... not ideal :smiley: